src/compatTest/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPluginTests.kt

```diff
@@ -206,6 +231,71 @@ class NexusPublishPluginTests {
         assertUploadedToStagingRepo("/org/example/sample/0.0.1/sample-0.0.1.jar")
     }
 
+    @Test
+    fun `publishes to two Nexus repositories`(@MethodScopeWiremockResolver.MethodScopedWiremockServer @Wiremock otherServer: WireMockServer) {
```

Comment:
this test isn't strictly necessary in order for us to feel confident about the changes that are now being made in this PR, but if you find it valuable I'm happy to leave it in.

Reply:
Yes, please.